<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddConsole</class>
 <widget class="QDialog" name="AddConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddConsole</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="brandLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Console brand</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="brandComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Console name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="consoleNameComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Problem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="consoleProblemText"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="clientHorizontalLayout">
       <item>
        <widget class="QCheckBox" name="existingClientCheckBox">
         <property name="text">
          <string>Existing client</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="clientComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="newClientCheckBox">
         <property name="text">
          <string>New client</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="newClientVerticalLayout">
         <item>
          <layout class="QHBoxLayout" name="firstNameHorizontalLayout">
           <item>
            <widget class="QLabel" name="firstName">
             <property name="text">
              <string>First name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="firstNameLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="nameHorizontalLayout">
           <item>
            <widget class="QLabel" name="name">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="phoneHorizontalLayout">
           <item>
            <widget class="QLabel" name="phone">
             <property name="text">
              <string>Phone number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="phoneLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="emailHorizontalLayout">
           <item>
            <widget class="QLabel" name="email">
             <property name="text">
              <string>Email address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailLine"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QPushButton" name="createConsoleButton">
           <property name="text">
            <string>Create console</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
